cypress/TestCases/FireboltCertification/LifecycleTestcases/LifecycleTerminated.feature

```diff
@@ -4,8 +4,8 @@ Feature: Lifecycle_Terminated
     Scenario Outline: Lifecycle 2.8 Terminate app in <state> State
         Given the environment has been set up for 'lifecycle' tests
         When 3rd party 'certification' app is launched with '<state>' state
-        Then '3rd party app' invokes the 'Firebolt' API to 'call lifecycle terminate'
-        Then '3rd party app' is in 'terminated' state
+        Then '3rd party app' transitions to state 'terminated'
+        Then '3rd party app' will stay in 'terminated' state
 
```

Comment:
Then '3rd party app' will stay in 'terminated' state
to be changed to 
Then '3rd party app' will be in 'terminated' state

Reply:
updated